source/MRMesh/MRObjectLabel.h

```diff
@@ -50,6 +50,12 @@ class MRMESH_CLASS ObjectLabel : public VisualObject
     MRMESH_API void setFontPath( const std::filesystem::path& pathToFont );
     const std::filesystem::path& getFontPath() const { return pathToFont_; }
 
+    /// set pivot point [0, 0] - [1, 1]
+    MRMESH_API void setPivotPoint( const Vector2f& pivotPoint );
```

Comment:
mb allow to have pivot out of [0,0]-[1,1]?

Reply:
and pls add getter for pivot point

Reply:
fixed

Reply:
pls add a more detailed comment about what pivot point do and what is default value of it

Reply:
added more comments